demonstrations/tutorial_trapped_ions.py

```diff
@@ -0,0 +1,1092 @@
+
+    qml.QubitUnitary(evolution(0, -np.pi / 2 / Omega), wires=0)
+    qml.QubitUnitary(evolution(np.pi / 2, np.pi / 2 / Omega), wires=0)
+    qml.QubitUnitary(evolution(0, np.pi / 2 / Omega), wires=0)
+    qml.QubitUnitary(evolution(np.pi / 2, np.pi / 2 / Omega), wires=0)
+    qml.QubitUnitary(evolution(0, np.pi / 2 / Omega), wires=0)
+
+    return qml.state()
+
+#For comparison, we use the built-in Hadamard on PennyLane
+@qml.qnode(dev)
+def hadamard(state):
+
+    if state == 1:
+        qml.PauliX(wires=0)
+
+    qml.Hadamard(wires=0)
+
+    return qml.state()
+
+#We confirm that the values given by both functions are the same up to numerical error
+print(np.isclose(1j * ion_hadamard(0), hadamard(0)))
+print(np.isclose(1j * ion_hadamard(1), hadamard(1)))
+
+##############################################################################
+# Where we note that the desired gate was obtained up to a global phase factor.
+# A similar excersise can be done for the T-gate:
+
+
+@qml.qnode(dev)
+def ion_Tgate(state):
+
+    if state == 1:
+        qml.PauliX(wires=0)
+
+    qml.QubitUnitary(evolution(0, -np.pi / 2 / Omega), wires=0)
+    qml.QubitUnitary(evolution(np.pi / 2, np.pi / 4 / Omega), wires=0)
+    qml.QubitUnitary(evolution(0, np.pi / 2 / Omega), wires=0)
+
+    return qml.state()
+
+
+@qml.qnode(dev)
+def tgate(state):
+
+    if state == 1:
+        qml.PauliX(wires=0)
+
+    qml.T(wires=0)
+
+    return qml.state()
+
+
+print(np.isclose(np.exp(1j * np.pi / 8) * ion_Tgate(0), tgate(0)))
+print(np.isclose(np.exp(1j * np.pi / 8) * ion_Tgate(1), tgate(1)))
+
+##############################################################################
+# This Pennylane code shows that we can obtain a Hadamard gate and a
+# T-gate using consecutive pulses with different times and phases. Namely,
+# to get a Hadamard gate, we need five pulses, all of them with duration
+# :math:`t=\frac{\pi}{2\Omega}`, where the second and the fourth pulse
+# have a phase of :math:`\pi/2`. The Hadamard and T gates together can be used to
+# implement any operation on a single qubit, to an arbitrary degree of approximation. We
+# see that timing and dephasing our laser pulses provide a versatile way
+# to manipulate single qubits.
+#
+# To get a better idea about how the duration
+# of the pulses affects the state that we generate, let us plot the probability
+# of obtaining the state :math:`\left\lvert e \right\rangle` against the
+# duration of the pulse for a fixed phase of :math:`\varphi = 0`.
+
+import matplotlib.pyplot as plt
+
+
+@qml.qnode(dev)
+def evolution_prob(t):
+
+    qml.QubitUnitary(evolution(0, t / Omega), wires=0)
+
+    return qml.probs(wires=0)
+
+
+t = np.linspace(0, 4 * np.pi, 101)
+s = [evolution_prob(i)[1].numpy() for i in t]
+
+fig1, ax1 = plt.subplots(figsize=(9, 6))
+
+ax1.plot(t, s, color="#9D2EC5")
+
+ax1.set(xlabel="time (in units of 1/Ω)", ylabel="Probability", title="Probability of measuring the excited state")
```

Comment:
```suggestion
ax1.set(
    xlabel="time (in units of 1/Ω)", 
    ylabel="Probability", 
    title="Probability of measuring the excited state"
)
```
Line was too long and made a horizontal scroll bar; we generally like to avoid those so that all the code is always visible